content/articles/launching-proppy.md

```diff
@@ -0,0 +1,35 @@
+Title: We are Wizards launch a new product
+Date: 2016-03-25
+Short_summary: After wrangling with proposals for a while we productized one of our internal tools.
+Category: Product
+Authors: Vincent, Tom
+
+*Reading time: ~3 minutes.*
+
+At We Are Wizards we send out a fair number of proposals. In our early
+days we developed a command line tool to generate consistent and
+error-free proposals - and this turned out to be super useful.
+
+We've taken our ramshackle tool and made it into a full product so
+others can benefit from this, too.
+
+<!-- PELICAN_END_SUMMARY -->
+
+When sending out proposals we identified a few pain points:
+
+* Keeping track of the proposal status in an email thread doesn't
+  really work.
+* PDFs aren't a great format for the wild number of devices people
+  are using these days.
+* Keeping cost table sums up to date is error prone without copying
+  from a spreadsheet.
+* Collaboration via Dropbox is painful.
```

Comment:
collaboration by sharing files (since it can be on dropbox, sugarsync, whatever)